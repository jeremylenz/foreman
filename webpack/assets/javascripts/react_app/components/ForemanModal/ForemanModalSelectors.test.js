import { testSelectorsSnapshotWithFixtures } from '../../common/testHelpers';
import {
  selectForemanModalsState,
  selectModalStateById,
} from './ForemanModalSelectors';

const state = {
  foremanModals: {
    myModal: { open: true },
    yourModal: { open: false },
  },
};

const fixtures = {
  'selects entire modals state': () => selectForemanModalsState(state),
  'selects specific modal by id': () => selectModalStateById(state, 'myModal'),
};

describe('ForemanModal selectors', () => {
  testSelectorsSnapshotWithFixtures(fixtures);
});
