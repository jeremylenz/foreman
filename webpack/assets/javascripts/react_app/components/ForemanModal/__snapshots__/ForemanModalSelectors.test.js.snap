// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForemanModal selectors selects entire modals state 1`] = `
Object {
  "myModal": Object {
    "open": true,
  },
  "yourModal": Object {
    "open": false,
  },
}
`;

exports[`ForemanModal selectors selects specific modal by id 1`] = `
Object {
  "open": true,
}
`;
